### ISSUE TITLE
On upgrade to 2.0.10, template output contains a <THYMELEAF_ROOT> element

### ISSUE THREAD

**Issue by ben3000** [thymeleaf/thymeleaf#34]
On output, my templates begin with:

```
<!DOCTYPE html>
<HTML><head></head><BODY><THYMELEAF_ROOT>
```

after upgrading Thymeleaf from 2.0.8 to 2.0.10.

If I revert to 2.0.8, the templates return to their normal output:

```
<!DOCTYPE html>
<html lang="en"><head>
    (header scripts and tags here)
</head><body>
```

**Comment by danielfernandez** [comment]
This is a bug. Thanks for reporting.

This affects LEGACYHTML5-mode templates under specific circumstances.

This has already been solved in source trunk and a 2.0.11-SNAPSHOT version has been uploaded including this fix. You can use this snapshot version if you want before 2.0.11 is released --probably next weekend-- with some additional minor bugfixing.

See http://www.thymeleaf.org/faq.html in order to learn how to use a snapshot version.

Regards,
Daniel

**Comment by ben3000** [comment]
Yep, this fixes it for me, thanks.
